cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -4413,6 +4413,15 @@
     </description>
   </property>
 
+  <!-- Authenticator Configuration -->
+  <property>
+    <name>remote.authenticator.extensions.dir</name>
```

Comment:
Is this supported in sandbox environment as well? If so, should we add this property in sandbox cdap-default as well?

Reply:
Resolved offline, should be fine for it to live in cdap-default for standalone.